jsonschema-core.xml

```diff
@@ -1394,87 +1412,179 @@
+                            then the JSON Pointer fragment URI for that schema relative to its
+                            parent's base URI would then identify the reference, while JSON Pointers
+                            to locations within the formerly embedded resource would become invalid.
                             <cref>
-                                How should an "$id" URI reference containing a fragment with other components
-                                be interpreted?  There are two cases:  when the other components match
-                                the current base URI and when they change the base URI.
+                                If the change regarding reference replacement noted in the previous
+                                CREF were to be implemented, the pointer behavior would be more
+                                consistent.  Really it is the pointers to deeper locations within
+                                embedded schemas which should be strongly discouraged and
+                                need not be supported.
                             </cref>
                         </t>
+                        <t>
+                            Examples of such non-canonical URIs, as well as the appropriate
+                            canonical URIs to use instead, are provided in section
+                            <xref target="idExamples" format="counter"></xref>.
+                        </t>
                     </section>
-                    <section title="Schema identification examples" anchor="idExamples">
-                        <figure>
-                            <preamble>
-                                Consider the following schema, which shows "$id" being used to identify
-                                the root schema, change the base URI for subschemas, and assign plain
-                                name fragments to subschemas:
-                            </preamble>
-                            <artwork>
+                </section>
+                <section title='Defining location-independent identifiers with "$anchor"'
+                         anchor="anchor">
+                    <t>
+                        Using JSON Pointer fragments requires knowledge of the structure of the schema.
+                        When writing schema documents with the intention to provide re-usable
+                        schemas, it may be preferable to use a plain name fragment that is not tied to
+                        any particular structural location.  This allows a subschema to be relocated
+                        without requiring JSON Pointer references to be updated.
+                    </t>
+                    <t>
+                        The "$anchor" keyword is used to specify such a fragment.
+                    </t>
+                    <t>
+                        If present, the value of this keyword MUST be a string, which MUST start with
```

Comment:
/this keyword/$anchor/
I think this is more in keeping with most of the document.

(My preference would be to change all instances of "this keyword" to "[keyword]", making the document more easy to scan and search for specific keyword requirements as a reference document.

Reply:
@Relequestual I'll do some of this, but at this stage I do not want to spend to much time fine-tuning wording.  If it's understandable, let's get this done.

Reply:
@Relequestual actually I went and looked and I'm really consistent about using "this keyword".  I don't want to chase it all over- you are welcome to do a stylistic revamp when you take on the next draft 😃